### PR TITLE
GitHub User Experience

### PR DESCRIPTION
**Pull request by PantherX**
I have updated templated and massively simplified them to ensure that it is easy to either report an issue or suggest a feature.
I have also included links to various sites where support can be obtained.
I have created a security policy and updated the readme file.

### CODE REVIEW

**Comment by shorttack** [comment]
How will mods here enter non-specific issues that do not fit into a specific template? [https://github.com/FoldingAtHome/fah-issues/pull/1481/commits/ee4b85b1af4ffc711b4a1097f7d2478cc5866178](url)

**Comment by PantherX** [comment]
@shorttack 
Can you please provide an example so I can work through it and/or modify the files to be fit for purpose?

**Comment by kskilling** [comment]
Looks good to me. Useful to ensure a standard set of information is provided by users for each issue logged.

File: .github/ISSUE_TEMPLATE/SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Security
+
+## Reporting a Bug in F@H Software
+
+To report security vulnerabilities, please email joseph(at)cauldrondevelopment(dot))com
```
Comment (MichaIng) [inline .github/ISSUE_TEMPLATE/SECURITY.md:5]: `(dot))com` should be `(dot)com` I guess? 🙂

**Comment by MichaIng** [comment]
@shorttack 
There is always the possibility to open a blank issue if none of the templates fit:
fahclient

File: .github/ISSUE_TEMPLATE/SECURITY.md (hunk 1 above)
Comment (PantherX) [reply .github/ISSUE_TEMPLATE/SECURITY.md]: Thanks for that... clearly, I need more than four eyes :sweat_smile:

**Pull request merged**